docs/tasks.md

```diff
@@ -502,6 +502,23 @@ spec:
       value: "http://google.com"
 ```
 
+Parameter declarations include default values which will be used if the parameter is not specified, for example to
+specify defaults for both string params and array params ([full example](../examples/v1beta1/taskruns/array-default.yaml)) :
+
+```yaml
+    params:
+      - name: flags
```

Comment:
This example is a bit confusing; I would expect most people to want to use only one parameter for flags and to supply some default values for that parameter (am I misunderstanding the example?). Also, could you provide slightly more realistic default values for flags? e.g. the example above has "arg1=foo" and "--randomflag".

Reply:
good point - ill update this